packages/workflow/src/workflow.ts

```diff
@@ -456,8 +452,6 @@ export async function startChild<T extends Workflow>(
 /**
  * Start a child Workflow execution
  *
- * **Override for Workflows that accept no arguments**.
```

Comment:
This is actually correct, so are some of the other overrides below

Reply:
All of the ones without `options`? Would it be more accurate to say "Override for when you don't need `options`"? In which case maybe we don't need to include, since that's clear from the signature, and they appear in order after the overrides that have `options`?

Reply:
All of the options are optional so the override is only for workflows that accept no arguments (see the generic type constraint: `T extends () => Promise<any>`)

Reply:
`args` is required if the workflow takes arguments

Reply:
Is the generic type constraint needed? This compiles:

![image](https://user-images.githubusercontent.com/251288/151652006-fcf52d01-511a-4e68-a42a-19857a5b6f28.png)

Reply:
What is this supposed to show? There's nothing here that references the workflow arguments.